frontend/js/response_chart.js

```diff
@@ -1,64 +1,49 @@
-$(function(){
-  if($('#responses_chart').length == 0) {
+$(function () {
+  if ($('#checks_chart').length == 0) {
     return;
   }
 
-  chart = c3.generate({
-    bindto: '#responses_chart',
-    size: {
-      height: 200
-    },
-    data: {
-      colors: {
-        'good response': '#41C24C',
-        'bad response': '#FD7466'
-      },
-      x: 'time',
-      xFormat: '%Y-%m-%d %H:%M:%S',
-      columns: [],
-      types: {
-        'good response': 'area-step',
-        'bad response': 'area-step'
+  var checkId = $('#checks_chart').data('id');
+
+  function drawDiagram(data) {
+    var parentContainer = $('#checks_chart');
+    parentContainer.html('');
+    var width = 100;
+    var legendScale = 20;
+    var height = parentContainer.height();
+    var log = data.log;
+    var amount = log.length;
+    var maxOfTimeouts = _.maxBy(log, item =>  { return item.timeout; } ).timeout;
```

Comment:
I know, one of the _latest_ firefoxes
but the browsers support is still poor
https://kangax.github.io/compat-table/es6/